tools/build_defs/apple/swift.bzl

```diff
@@ -382,8 +424,11 @@ def _swift_library_impl(ctx):
   ctx.file_action(output=swiftc_output_map_file, content=swiftc_output_map.to_json())
 
   args = _swift_xcrun_args(ctx) + ["swiftc"] + swiftc_args(ctx)
+  args += sorted(set(["-I" + d.dirname for d in dep_modules + [output_module]]))
+
   args += [
-      "-I" + output_module.dirname,
+      "-Xcc", "-fmodule-map-file=" + unextended_modulemap.path,
```

Comment:
If I understand this right, instead of loading a module map generated by each objc_ dependency, you generate a huge map, that includes all deps right in this target, correct?